spec.md

```diff
@@ -1,4 +1,4 @@
-# CloudEvents - Version 0.2
+﻿# CloudEvents - Version 0.2
```

Comment:
weird

Reply:
The updated version adds a extra whitespace to the end of 0.2.